### PR TITLE
Renamed fields

### PR DESCRIPTION
**Pull request by himanshu007-creator**
#2053

### CODE REVIEW

**Comment by EdColeman** [comment]
There looks to be additional references that you missed.  Please fix and then run
`mvn clean verify -DskipITs`
To build locally - once you have a clean build you can push your changes and we can try the QA tests again. (See: https://accumulo.apache.org/how-to-contribute/#example-contribution-workflow) for more information.

**Review by EdColeman — CHANGES_REQUESTED** [review]
QA test failed - additional references need to be updated.

**Comment by himanshu007-creator** [comment]
@EdColeman  HI, i dont have system requirements for setting up the project, can you suggest a way out? I thought of it to be an issue just related to renaming fields , nothing else.

**Comment by brianloss** [comment]
> As suggested from @brianloss on another issue: If you have or can install Docker, then you could try building from a Docker container. Something like [maven](https://hub.docker.com/_/maven) probably work if you volume mount your source directory. Either that or a linux vm (either local or hosted) are options.

I played around a little bit on a Windows VM, and discovered that WSL 2 works quite well if that's an option for you. I installed the Ubuntu 20.04 distro and had to install JDK, Maven, make, and g++ and I was able to do a build w/o tests. Docker for Windows made me install WSL (there is a hyper-v option too, but didn't work for me since I was already in a VM I think), so no benefit to Docker in that case. If you want Docker for other reasons (e.g., not installing tools), unfortunately the maven images don't quite work since they don't include the necessary tools to build the native libs. It's pretty easy to build an image on top of the maven ones, though. For example, I had the following Dockerfile to build an image:
```dockerfile
FROM maven:3-jdk-11-slim
RUN apt-get update && apt install -y g++ make && rm -rf /var/lib/apt/lists/*
```
With my custom image, I was able to do a full build (again, skipping test execution). I just followed the instructions on the [maven](https://hub.docker.com/_/maven) image page for running and also to reuse the m2 repository (you don't want to have to download all of the dependencies every time).

**Comment by himanshu007-creator** [comment]
> > As suggested from @brianloss on another issue: If you have or can install Docker, then you could try building from a Docker container. Something like [maven](https://hub.docker.com/_/maven) probably work if you volume mount your source directory. Either that or a linux vm (either local or hosted) are options.
> 
> I played around a little bit on a Windows VM, and discovered that WSL 2 works quite well if that's an option for you. I installed the Ubuntu 20.04 distro and had to install JDK, Maven, make, and g++ and I was able to do a build w/o tests. Docker for Windows made me install WSL (there is a hyper-v option too, but didn't work for me since I was already in a VM I think), so no benefit to Docker in that case. If you want Docker for other reasons (e.g., not installing tools), unfortunately the maven images don't quite work since they don't include the necessary tools to build the native libs. It's pretty easy to build an image on top of the maven ones, though. For example, I had the following Dockerfile to build an image:
> 
> ```dockerfile
> FROM maven:3-jdk-11-slim
> RUN apt-get update && apt install -y g++ make && rm -rf /var/lib/apt/lists/*
> ```
> 
> With my custom image, I was able to do a full build (again, skipping test execution). I just followed the instructions on the [maven](https://hub.docker.com/_/maven) image page for running and also to reuse the m2 repository (you don't want to have to download all of the dependencies every time).

i am sorry, but i wont be able to do that, i would appreciate if you can make the required changes in this PR, ill commit the changes and then its all settled

**Comment by EdColeman** [comment]
The lines that need changed are:

[line 366](https://github.com/apache/accumulo/blob/4f35e6aff1763d4b19d1e93036260094b47cc589/server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java#L366)

[line 370](https://github.com/apache/accumulo/blob/4f35e6aff1763d4b19d1e93036260094b47cc589/server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java#L370)

[line 862](https://github.com/apache/accumulo/blob/4f35e6aff1763d4b19d1e93036260094b47cc589/server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java#L862)

You should at least be able to compile in your ide without errors.  Please make the changes on your branch and the push the changes to github.

File: server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java [hunk 1]
```diff
@@ -98,7 +98,7 @@
   private final ExecutorService assignmentPool;
   private final ExecutorService assignMetaDataPool;
   private final ExecutorService summaryRetrievalPool;
-  private final ExecutorService summaryParitionPool;
+  private final ExecutorService summaryParititionPool;
```
Comment (EdColeman) [inline server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java:101]: This should be summaryPartitionPool

**Review by EdColeman — CHANGES_REQUESTED** [review]

File: server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java (hunk 1 above)
Comment (himanshu007-creator) [reply server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServerResourceManager.java]: done

**Comment by EdColeman** [comment]
Fix #2053 Fix typo - rename summaryParitionPool to summaryPartitionPool

**Comment by EdColeman** [comment]
Thanks for the contribution @himanshu007-creator  and welcome to the Apache Accumulo community. If you would like to be listed as a contributor on Accumulo's people page please make a website PR for people.md. https://github.com/apache/accumulo-website/edit/main/pages/people.md
If you intend to make more contributions, please consider subscribing to the dev list and introducing yourself. Also, you can use the dev list or slack to reach out if you have questions about anything. https://accumulo.apache.org/contact-us/

**Pull request merged**